start.sh

```diff
@@ -35,6 +35,13 @@ export CATALINA_OPTS="-server -Xms1g -Xmx2g -Djava.awt.headless=true -XX:+UseCom
 echo JAVA_OPTS:
 echo $JAVA_OPTS
 
+# If the first argument is -debugger, then start Tomcat in debug mode on port 8000
```

Comment:
Its something I use a lot. Specially in tools like lesson, where I have to restart tomcat everytime. Its easier to just add break points in a debbuger.